src/Events/Admin/Notice/Rest_Api.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Shows an admin notice when our REST API endpoints are not available.
+ *
+ * @since
+ *
+ * @package TEC\Events\Admin\Notice
+ */
+
+namespace TEC\Events\Admin\Notice;
+
+use Tribe\Events\Views\V2\Rest_Endpoint as V2;
+
+/**
+ * Class Rest_Api
+ *
+ * Shows an admin notice when our REST API endpoints are not available.
+ *
+ * @since
+ */
+class Rest_Api {
+
+	/**
+	 * Notice Slug on the user options
+	 *
+	 * @since
+	 *
+	 * @var string
+	 */
+	private $slug = 'events-rest-api-notice';
+
+	/**
+	 * Blocked endpoint.
+	 *
+	 * @since
+	 *
+	 * @var string
+	 */
+	private $blocked_endpoint = '';
+
+	/**
+	 * Constructor.
+	 *
+	 * @since
+	 *
+	 * @return void
+	 */
+	public function hook(): void {
+		$slug = $this->slug;
+
+		tribe_notice(
+			$slug,
+			[ $this, 'notice' ],
+			[
+				'type'    => 'error',
+				'dismiss' => 1,
+				'wrap'    => 'p',
+			],
+			[ $this, 'should_display' ]
+		);
+	}
+
+	/**
+	 * Checks if we are in a TEC page or in the main Dashboard.
+	 *
+	 * @since
+	 *
+	 * @return boolean
+	 */
+	public function should_display(): bool {
+		global $pagenow;
+
+		if ( tribe( 'admin.helpers' )->is_screen() || 'index.php' === $pagenow ) {
+			return $this->is_rest_api_blocked() ? true : false;
+		}
+
+		return false;
+	}
+
+	/**
+	 * Checks if our endpoints are accessible.
+	 *
+	 * @since
+	 *
+	 * @return boolean
+	 */
+	public function is_rest_api_blocked(): bool {
+
+		$v1_api    = new \Tribe__Events__REST__V1__Main();
+		$event_api = get_rest_url( null, $v1_api->get_events_route_namespace() );
+		$response  = wp_remote_get( $event_api );
+		if ( is_wp_error( $response ) || wp_remote_retrieve_response_code( $response ) !== 200 ) {
+			$this->blocked_endpoint = $event_api;
+			return true;
+		}
+
+		$views_api = get_rest_url( null, V2::ROOT_NAMESPACE );
+		$response  = wp_remote_get( $views_api );
+		if ( is_wp_error( $response ) || wp_remote_retrieve_response_code( $response ) !== 200 ) {
+			$this->blocked_endpoint = $views_api;
+			return true;
+		}
+
+		return false;
+	}
+
+	/**
+	 * HTML for the notice when we have blocked REST API endpoints.
+	 *
+	 * @since
+	 *
+	 * @return false|string
+	 */
+	public function notice() {
+		if ( ! current_user_can( 'activate_plugins' ) ) {
+			return false;
+		}
+
+		$text = [];
+		// Translators: %s is the "Warning" word in bold.
+		$text[] = sprintf( __( '%s : The Events Calendar REST API endpoints are not accessible! This may be due to a server configuration or another plugin blocking access to the REST API.', 'the-events-calendar' ), '<strong>' . __( 'Warning', 'the-events-calendar' ) . '</strong>' );
```

Comment:
```suggestion
		/* Translators: %1$s and %2$s - opening and closing strong tags, respectively. */
				$text[] = sprintf(
			__( '%1$sWarning%2$s: The Events Calendar REST API endpoints are not accessible! This may be due to a server configuration or another plugin blocking access to the REST API.', 'the-events-calendar' ),
			'<strong>',
			'</strong>'
		);
```
Whenever possible, keep strings together in translations for better context. In this case we can just use placeholders for the tags.

Reply:
Otherwise: the "Warning" gets translated out of context - which could result in a poorer translation when combined.

Reply:
In my experience, using placeholders for html tags in translation strings is a bad practice because it is prone to user errors. I have encountered un-closed html tags numerous times that broke entire pages, because of a user messing up the placeholders. 

IMO, we shouldn't expect users to know what HTML tags are and translation strings should not be trusted. Thus keeping html out of `__()` is the safest approach. Still, if this is something allowed here, then I will take that shortcut in the future.

As for better context, we have the `_x()` functions whenever needed.